stories/FileTree.stories.tsx

```diff
@@ -60,10 +69,90 @@ const version = {
         filename: 'styles.css',
         path: 'styles.css',
       },
+      'jquery-ui': {
+        ...fakeVersionEntry,
+        depth: 0,
+        filename: 'jquery-ui',
+        mime_category: 'directory' as VersionEntryType,
+        path: 'jquery-ui',
+      },
+      'jquery-ui/css': {
+        ...fakeVersionEntry,
+        depth: 1,
+        filename: 'css',
+        mime_category: 'directory' as VersionEntryType,
+        path: 'jquery-ui/css',
+      },
+      'jquery-ui/js': {
+        ...fakeVersionEntry,
+        depth: 1,
+        filename: 'js',
+        mime_category: 'directory' as VersionEntryType,
+        path: 'jquery-ui/js',
+      },
+      'jquery-ui/js/jquery-1.7.1.min.js': {
+        ...fakeVersionEntry,
+        depth: 2,
+        filename: 'jquery-1.7.1.min.js',
+        mime_category: 'text' as VersionEntryType,
+        path: 'jquery-ui/js/jquery-1.7.1.min.js',
+      },
+      'jquery-ui/js/jquery-ui-1.8.16.custom.min.js': {
+        ...fakeVersionEntry,
+        depth: 2,
+        filename: 'jquery-ui-1.8.16.custom.min.js',
+        mime_category: 'text' as VersionEntryType,
+        path: 'jquery-ui/js/jquery-ui-1.8.16.custom.min.js',
+      },
+      lib: {
+        ...fakeVersionEntry,
+        depth: 0,
+        filename: 'lib',
+        mime_category: 'directory' as VersionEntryType,
+        path: 'lib',
+      },
+      'lib/adblockplus.js': {
+        ...fakeVersionEntry,
+        depth: 1,
+        filename: 'adblockplus.js',
+        mime_category: 'text' as VersionEntryType,
+        path: 'lib/adblockplus.js',
+      },
+      'lib/compat.js': {
+        ...fakeVersionEntry,
+        depth: 1,
+        filename: 'compat.js',
+        mime_category: 'text' as VersionEntryType,
```

Comment:
I won't comment on all of them but there are several others that you can remove.